components/Home/Home.jsx

```diff
@@ -78,8 +71,8 @@ export default () => {
           name={cycleDetails && cycleDetails[cycles.selectedCycleIndex].name}
           subtext={cycleDetails && cycleDetails[cycles.selectedCycleIndex].muscleGroups}
           color={cycleDetails && cycleDetails[cycles.selectedCycleIndex].color}
-          leftPress={() => { dispatch({ type: DECREMENT_SELECTED_CYCLE_INDEX, cycleLength: cycleDetails.length }); }}
-          rightPress={() => { dispatch({ type: INCREMENT_SELECTED_CYCLE_INDEX, cycleLength: cycleDetails.length }); }}
+          leftPress={() => { dispatch(actions.cycles.decrementSelectedCycleIndex(cycleDetails.length)); }}
+          rightPress={() => { dispatch(actions.cycles.incrementSelectedCycleIndex(cycleDetails.length)); }}
           id={cycleDetails && selectedCycle.workouts[cycles.selectedCycleIndex]}
         />
         <WorkoutSwipeList items={workoutList} style={{ marginLeft: '10%' }} />
```

Comment:
The only redux-y thing that doesnt seem perfect yet is the recent workouts on the home page. If you finish a workout, I feel like it should be put at the front, and also the n/a days ago is a bit odd. 
<img width="335" alt="Screen Shot 2021-01-12 at 1 34 13 PM" src="https://user-images.githubusercontent.com/15805074/104377144-dd757c00-54da-11eb-89bc-6ef950dd4c15.png">

Also, ^^thats not something that needs to be addressed in this branch, I just stumbled across it during review.

Reply:
Yeah that's something that should be a part of Rohith's task updating Home to use Redux stuff

Reply:
this is complete!